observation_portal/sciapplications/models.py

```diff
@@ -494,6 +499,8 @@ class ScienceApplicationUserReview(models.Model):
 
     grade = models.DecimalField(blank=True, null=True, default=None, max_digits=4, decimal_places=2)
 
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
In other models in this project we use a field name of `created` for this rather than `created_at`. And you might want to include `modified` too as an `auto_now=True` if that is useful?